github-extension/src/browser/github-frontend-module.ts

```diff
@@ -11,9 +11,21 @@
 
 import { ContainerModule } from "inversify";
 import { GithubService, githubKeyServicePath } from '../common/github-service';
-import { WebSocketConnectionProvider } from "@theia/core/lib/browser";
+import { WebSocketConnectionProvider, FrontendApplicationContribution } from "@theia/core/lib/browser";
+import { GitHubQuickOpenService } from "./github-quick-open-service";
+import { CommandContribution } from "@theia/core";
+import { GitHubCommandHandlers } from "./github-commands";
+import { GitHubFrontendContribution } from "./github-frontend-contribution";
+import { GitHubProperties } from "./github-properties";
 
 export default new ContainerModule(bind => {
+    bind(GitHubProperties).toSelf().inSingletonScope();
+    bind(FrontendApplicationContribution).toDynamicValue(c => c.container.get(GitHubProperties));
```

Comment:
fyi: such bindings can be simplified like `bind(FrontendApplicationContribution).toService(GitHubProperties)` since `toService` uses `toDynamicValue` under the hood

Reply:
Thanks, I'll fix this